android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/dao/ActivityListDAO.java

```diff
@@ -33,7 +34,7 @@
 /**
  * Created by TheMDP on 12/29/17.
  */
-
+@BridgeStudyScope // TODO: @liujoshua consider scoping/namespacing to participant 2018/10/09
 public class ActivityListDAO extends SharedPreferencesJsonDAO {
```

Comment:
Side note, we should disable this class all together.  It will just take up memory that we don't use in mPower 2 because we use Room.